Scripts/SwiftLint.sh

```diff
@@ -26,8 +26,19 @@ if [ -z "$selected_path" ]; then
 	exit 1
 fi
 
+# copy the generated output to a temp dir and strip the "swiftlint:disable:all"
 if [ "$key" = "templates_generated" ]; then
-	find "${PROJECT_DIR}/${selected_path}" -name "*.swift" -exec ${PROJECT_DIR}/Scripts/swiftlint_no_disable_all.sh {} \;
+	temp_dir=`mktemp -d -t SwiftGen`
+
+	for f in `find "${PROJECT_DIR}/${selected_path}" -name '*.swift'`; do
+		temp_file="${temp_dir}${f#*"$PROJECT_DIR"}"
+		mkdir -p $(dirname "$temp_file")
+		cat "$f" | sed s@"swiftlint:disable all"@" --"@ > "$temp_file"
```

Comment:
`cat | sed` is suboptimal as it spawns two processes. `sed` can take the input file as argument instead.

Reply:
Btw I think that was one reason for the performance issue (spawning too many processes) so that change might improve that even further

Reply:
No the performance issue is mainly invoking `swiftlint` on a per-file basis, it has to spawn, parse it's own config, etc... `swiftlint` is much faster if you give it a directory to parse.